packages/autocomplete-core/src/utils/isChrome.ts

```diff
@@ -0,0 +1,16 @@
+const chromeRegex = /(?:crmo|crios|chrome)\/v?([\w.]+)/i;
+const edgeRegex = /edg(?:e|ios|a)?\/([\w.]+)/i;
+const samsungBrowserRegex = /samsungbrowser\/([\w.]+)/i;
+const chromiumRegex = /chromium[/ ]([-\w.]+)/i;
+
+export function isChrome(userAgent: string) {
+  return (
+    userAgent &&
+    Boolean(
+      userAgent.match(chromeRegex) &&
+        !userAgent.match(edgeRegex) &&
+        !userAgent.match(samsungBrowserRegex) &&
+        !userAgent.match(chromiumRegex)
+    )
+  );
+}
```

Comment:
This ensures the functions always returns a boolean.

```suggestion
export function isChrome(userAgent: string) {
  return Boolean(
    userAgent &&
      userAgent.match(chromeRegex) &&
      !userAgent.match(edgeRegex) &&
      !userAgent.match(samsungBrowserRegex) &&
      !userAgent.match(chromiumRegex)
  );
}
```